content/configuration/externals.md

```diff
@@ -25,18 +29,62 @@ contributors:
 
 **webpack.config.js**
 
-```js
+```javascript
 externals: {
   jquery: 'jQuery'
 }
 ```
 
 This leaves any dependant modules unchanged, i.e. the code shown below will still work:
 
-```js
+```javascript
 import $ from 'jquery';
 
 $('.my-element').animate(...);
 ```
 
-?> TODO: Add more details on the various ways of doing this using an `object`, `function`, etc. Also add how this is connected to [`output.libraryTarget`](/configuration/output#output-librarytarget) and maybe how it's useful when building libraries if this isn't already evident.
+T> __consumer__ here is any end user application that includes the library that you have bundled using webpack.
+
+## Usage
```

Comment:
Why is this heading here? It seems out of place because `externals` is also on `##`.